backend/controllers/auth.js

```diff
@@ -1,8 +1,14 @@
 import authService from '../services/auth.js';
 
 export const login = (req, res) => {
-  const response = authService.login();
-  res.json({ message: response });
+  const { email, password } = req.body;
+
+  const token = authService.login(email, password);
+
+  if (token) {
+    res.status(200).json({ token });
```

Comment:
about statuses
check this package [http-statuses](https://www.npmjs.com/package/http-status-codes)